addons/binding/org.openhab.binding.solarlog/build.properties

```diff
@@ -0,0 +1,6 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/
```

Comment:
Please add the about.html to this file and make sure that it contains a new line at the end.